packages/vue-components/src/Tab.vue

```diff
@@ -3,9 +3,14 @@
     <div
       v-show="show"
       role="tabpanel"
-      class="tab-pane active"
+      class="tab-pane active printable-tab-pane d-print-block"
       :class="{hide:!show}"
     >
+      <div class="nav-tabs printable-tab-header d-none d-print-flex">
```

Comment:
Thanks!

Just one question: could we merge this with the existing `header` below? similar to what you did for `TabGroup`

Reply:
Yup it's possible. Thanks for the suggestion and I've made the changes accordingly :)